src/Racing.cpp

```diff
@@ -1,8 +1,6 @@
 #include "Racing.hpp"
-
 #include "db/Database.hpp"
 #include "servers/CNShardServer.hpp"
-
 #include "NPCManager.hpp"
 #include "PlayerManager.hpp"
 #include "Missions.hpp"
```

Comment:
Our headers are organized into a hierarchy to avoid things like cyclic inclusion. These spaces exist to indicate that, so please don't remove them.